src/client/Microsoft.Identity.Client/AppConfig/ConfidentialClientApplicationBuilder.cs

```diff
@@ -79,8 +79,10 @@ public static ConfidentialClientApplicationBuilder Create(string clientId)
         /// Sets the certificate associated with the application
         /// </summary>
         /// <param name="certificate">The X509 certificate used as credentials to prove the identity of the application to Azure AD.</param>
+        /// <param name="withSendX5C"> Specifies if the x5c claim (public key of the certificate) should be sent to the STS for Subject Name and 
+        /// Issuer certificate authentication. The default is <c>false</c></param>
         /// <remarks>You should use certificates with a private key size of at least 2048 bytes. Future versions of this library might reject certificates with smaller keys. </remarks>
-        public ConfidentialClientApplicationBuilder WithCertificate(X509Certificate2 certificate)
+        public ConfidentialClientApplicationBuilder WithCertificate(X509Certificate2 certificate, bool withSendX5C = true)
```

Comment:
This does nothing, the param isn't used anywhere!

Reply:
Removed